src/python/CRABClient/Commands/report2.py

```diff
@@ -443,9 +443,7 @@ def getDBSPublicationInfo(self, outputDatasets):
         for outputDataset in outputDatasets:
             res['outputDatasets'][outputDataset] = {'lumis': {}, 'numEvents': 0}
             try:
-                dbs = DBSReader("https://cmsweb.cern.ch/dbs/prod/phys03/DBSReader",
-                                cfg_dict = {"cert": self.proxyfilename, "key": self.proxyfilename,
-                                            "logger": self.logger, "pycurl" : True}) #We can only publish here with DBS3
+                dbs = DBSReader("https://cmsweb.cern.ch/dbs/prod/phys03/DBSReader")
```

Comment:
Let's pass to DBS the proxyfilename for cert and key, it makes me nervous to let the DBS client figure them out by himself